packages/discovery-react-components/src/components/SearchFacets/components/FacetsGroups/CollapsibleFacetsGroup.tsx

```diff
@@ -152,6 +155,7 @@ export const CollapsibleFacetsGroup: FC<CollapsibleFacetsGroupProps> = ({
           {facetsLabel}
           {shouldDisplayClearButton && (
             <ListBox.Selection
+              id="search-facets-clear-listbox"
```

Comment:
this isn't unique to the page as there can be multiple `CollapsibleFacetsGroup`s on the page. If you want to assign identifiers, i would make sure to define a prefix controlled at the top-level component (`SearchFacets`) so users can override it as needed